contents/handbook/small-teams/monitoring/index.mdx

```diff
@@ -1,34 +1,41 @@
 ---
-title: Monitoring Team
```

Comment:
`small-teams/monitoring` path has to be `small-teams/replay`, can push a fix if that's intended.

Reply:
Yeah theres way more needed if we want to do this. Just not doing all the work until I get final agreement on the name 😅